.github/workflows/release.yml

```
@@ -41,10 +44,17 @@ jobs:
           VERSION="${{ github.event.inputs.version }}"
         else
-          # Auto-determine version based on commit messages
-          if git log --format=%s -n 1 | grep -q "BREAKING CHANGE\|feat!"; then
+          # Auto-determine version based on commit messages since last tag
+          LAST_TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo "")
+          if [ -n "$LAST_TAG" ]; then
+            COMMIT_RANGE="$LAST_TAG..HEAD"
+          else
+            COMMIT_RANGE="HEAD~10..HEAD"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)